account_fiscal_position_rule_sale/__openerp__.py

```diff
@@ -23,10 +23,8 @@
 
 {
     'name': 'Account Fiscal Position Rule Sale',
-    'version': '1.1.1',
+    'version': '1.1',
```

Comment:
You can bump it to 1.0, as this is the first version for 8.0. According last discussions, this can be 8.0.1.0